system-int-test/src/cmr/system_int_test/utils/search_util.clj

```diff
@@ -854,3 +856,37 @@
                  :body (json/generate-string {:scroll_id scroll-id})
                  :throw-exceptions true
                  :connection-manager (s/conn-mgr)})))
+
+(defn remove-duplicate-points
+  [{:keys [points]}]
+  (->> (pv/points->rounded-point-map points)
+       vals
+       (remove #(> (count %) 1))
+       (map reverse)))
+
+(defn- depointify
+  [point]
+  [(.lon point) (.lat point)])
+
+(defn make-excessive-points-without-dups
+  "Returns a list of n-ish unique points."
```

Comment:
Would there be a way to guarantee N points?

Reply:
I could make it continue adding points until the count was exactly n, instead of just make n points and removing the duplicates, but I didn't think the test needed that level of accuracty.